# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/url/a-element-origin.html.ini
# Commit: 9ad299b5161b
# Full Hash: 9ad299b5161b0cedf852d56365310de499f94043
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2018-11-14 22:12:17
# Regressor Bug: 1495313
# File Overlap Count: 1
# Description:
#   Bug 1495313 - Allow empty username in URLs r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D11254
# ==============================================================================

diff -r 6293789f6e9c -r 9ad299b5161b testing/web-platform/meta/url/a-element-origin.html.ini
--- a/testing/web-platform/meta/url/a-element-origin.html.ini	Wed Nov 14 14:27:58 2018 +0000
+++ b/testing/web-platform/meta/url/a-element-origin.html.ini	Wed Nov 14 15:24:21 2018 +0000
@@ -1,10 +1,4 @@
 [a-element-origin.html]
-  [Parsing origin: <https://:@test> against <about:blank>]
-    expected: FAIL
-
-  [Parsing origin: <http://::@c@d:2> against <http://example.org/foo/bar>]
-    expected: FAIL
-
   [Parsing origin: <gopher:/example.com/> against <http://example.org/foo/bar>]
     expected: FAIL
 
@@ -41,15 +35,6 @@
   [Parsing origin: <http::b@www.example.com> against <about:blank>]
     expected: FAIL
 
-  [Parsing origin: <http:/:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
-  [Parsing origin: <http://:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
-  [Parsing origin: <http://:@www.example.com> against <about:blank>]
-    expected: FAIL
-
   [Parsing origin: <i> against <sc:/pa/pa>]
     expected: FAIL
 